packages/steps/src/step.vue

```diff
@@ -173,6 +173,10 @@ export default {
   mounted() {
     const unwatch = this.$watch('index', val => {
       this.$watch('$parent.active', this.updateStatus, { immediate: true });
+      this.$watch('$parent.processStatus', () => {
```

Comment:
This part should be removed.

Rebase your changes on latest `dev` branch. Do not include changes other than those committed by you.